astro_2.0/components/Feed/components/FeedFilter/FeedFilter.tsx

```diff
@@ -10,8 +10,8 @@ import React, {
 import { PAGE_LAYOUT_ID } from 'constants/common';
 
 import { Icon } from 'components/Icon';
-import { RadioGroup } from 'astro_2.0/components/inputs/radio/RadioGroup';
-import { Radio, RadioProps } from 'astro_2.0/components/inputs/radio/Radio';
+import { RadioGroup } from 'astro_2.0/components/inputs/Radio/RadioGroup';
```

Comment:
I consider there is something wrong with the path. Currently radio is starting from the lowercase letter and I believe it's something that should be preserved